src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -559,28 +578,15 @@ export default {
 			// Update Lend Filter Exp CASH-545
 			this.getLendFilterExpVersion();
 		},
-		initializeMonthlyGoodPromo() {
+		async initializeMonthlyGoodPromo() {
```

Comment:
It doesn't look like this method needs `async`

Reply:
Yes, this looks like it will work without `async`